Riot/Modules/Analytics/Analytics.swift

```diff
@@ -149,17 +150,30 @@ import AnalyticsEvents
     
     /// Identify (pseudonymously) any future events with the ID from the analytics account data settings.
     /// - Parameter settings: The settings to use for identification. The ID must be set *before* calling this method.
-    private func identify(with settings: AnalyticsSettings) {
+    /// - Parameter userProperties: Any user properties that should be set for creating cohorts etc.
+    private func identify(with settings: AnalyticsSettings, and userProperties: AnalyticsEvent.UserProperties) {
         guard let id = settings.id else {
             MXLog.error("[Analytics] identify(with:) called before an ID has been generated.")
             return
         }
         
-        client.identify(id: id)
+        client.identify(id: id, userProperties: userProperties)
         MXLog.debug("[Analytics] Identified.")
         RiotSettings.shared.isIdentifiedForAnalytics = true
     }
     
+    /// Returns the user properties for use when identifying a session.
+    /// - Parameter session: The session to gather any user properties from.
+    /// - Returns: The properties to be set.
+    private func makeUserProperties(for session: MXSession) -> AnalyticsEvent.UserProperties {
+        var useCaseSelection: AnalyticsEvent.UserProperties.FtueUseCaseSelection?
```

Comment:
It took me a while to understand what `useCase` means in this context. It seems we use it both for architectural components (e.g. `OnboardingUseCase`) as well as a specific feature (i.e. use case = reason to use element). Is this correct?

Reply:
Ohhhh, thanks for pointing this out. I meant to rename `OnboardingUseCase` to `OnboardingUseCaseScreen`/`OnboardingUseCasePicker`/`OnboardingUseCaseView` after I created it. That's the screen where the `useCase` is selected.

It's annoying that the analytics schema has this as UseCaseSelection and not just UseCase. Do you think it would be clearer rename all occurances of `useCase` to `useCaseSelection` for consistency?

Reply:
I think my confusion was broader, as the term "use case" itself (even as "use case selection"), refers to something [quite generic](https://en.wikipedia.org/wiki/Use_case) wherease the FTUE means a more specific meaning, something like "what is your reason to use element". So to push it to an extereme, this would be `FTUEUseCaseUseCase` 😅 . Obviously I'm not suggesting we use that name, I was just trying to understand that distinction.

Reply:
For instance looking at [this type](https://github.com/vector-im/element-ios/pull/5591/files#diff-82a4c0953db3f6f8d8c9fd42781e36b097c527c283a477c4823143fe3aa5da04R177) `UserSessionProperties.UseCase` on its own, I would not know what it refers to without a wider context

Reply:
Understood, I'll have a think about this (Definition 2 in that link seems entirely new to me), but will go ahead and merge this PR 👍